pkg/new-ui/v1beta1/frontend/src/kubeflow.css

```diff
@@ -0,0 +1,9 @@
+body {
```

Comment:
This was an effort, started while developing the Central Dashboard, to expose a common css so that all of the kf apps to have a common theme https://github.com/kubeflow/kubeflow/pull/3115.

Yep, Katib UI is using this css file